src/core/display/DisplayObject.js

```diff
@@ -431,6 +431,25 @@ DisplayObject.prototype.generateTexture = function (renderer, scaleMode, resolut
 };
 
 /**
+ * Set the parent Container of this DisplayObject
+ *
+ * @param container {Container} The Container to add this DisplayObject to
+ * @return {DisplayObject} This DisplayObject
+ */
+DisplayObject.prototype.setParent = function (container)
+{
+    if (!container || !container.addChild)
+    {
+        throw new Error('setParent: Argument must be a Container');
+    }
+    else
+    {
+        container.addChild(this);
+        return this;
```

Comment:
It is the way it is now after a discussion on a previous issue and other chats outside it. Right now we will likely keep it the way it is because the community felt that this is more common:

```js
var container = new PIXI.Container(),
    child = container.addChild(new PIXI.Sprite);
```

than this:

```js
var container = new PIXI.Container().addChild(sprite);
```

Here is the issue for reference: https://github.com/pixijs/pixi.js/issues/299